packages/atomizer/src/rules.js

```diff
@@ -3300,6 +3300,74 @@ module.exports = [
     },
     /**
     ==================================================================
+    SCROLLBAR-COLOR
+    ==================================================================
+    */
+    {
+        type: 'pattern',
+        name: 'Scrollbar color',
+        matcher: 'Sbc',
+        allowParamToValue: false,
+        styles: {
+            'scrollbar-color': '$0 $1',
+        },
+        arguments: [
+            {
+                a: 'auto',
+                ...colors,
+            },
+            {
+                a: 'auto',
+                ...colors,
+            },
+        ],
+    },
+    /**
+    ==================================================================
+    SCROLLBAR-GUTTER
+    ==================================================================
+    */
+    {
+        type: 'pattern',
+        name: 'Scrollbar gutter',
+        matcher: 'Sbg',
+        allowParamToValue: false,
+        styles: {
+            'scrollbar-gutter': '$0 $1',
+        },
+        arguments: [
+            {
+                a: 'auto',
+                s: 'stable',
+            },
+            {
+                be: 'both-edges',
```

Comment:
@carriemorrison I'm not sure if this `both-edges` value applies to both `auto` and `stable` or just `stable`. If its the latter, I could just modify this rule to support a `sbe` value which maps to "stable both-edges".

The spec for reference: https://www.w3.org/TR/2021/WD-css-overflow-3-20211223/#propdef-scrollbar-gutter